src/main.cpp

```
@@ -539,12 +541,27 @@ int main (int argc, char **argv)
         << "O: " << opt.O << std::endl
         << "E: " << opt.E << std::endl;
 
+    std::cerr << std::endl << "Verifying parameters:" << std::endl;
+    try
+    {
+	map_param.verify();
+    }
+    catch(std::exception& e)
```

This is too broad: If you catch `std::exception`, you catch every type of exception of the standard library. If on error occurred unrelated to the verification, you mask that error. Introducing a common base class for the verification exceptions would allow you to be more specific here.